### PR TITLE
[Ready] New adv surgery tools!

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
[Changelogs]
Adds a new set of surgery tools that are not as good as augmented tools
New node that holds the tools as well as drapes and the basic medkit

[why]
Medkits are for Medibot army
drapes and tools so that people don't need augmented tools to be faster.
Augmented tools are still faster and will always be with you, not taking space in your bag. Its a nice middle ground

### CODE REVIEW

File: code/modules/research/designs/medical_designs.dm [hunk 1]
```diff
@@ -44,6 +44,16 @@
 	category = list("Medical Designs")
 	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL
 
+/datum/design/medicalkit
+	name = "Medical Kit"
```
Comment (Ragolution) [inline code/modules/research/designs/medical_designs.dm]: Empty Medkit

Comment (Ragolution) [inline code/modules/research/designs/medical_designs.dm]:
```suggestion
	name = "Empty Medkit"
```

So it doesn't get confused with full ones.

Comment (Ragolution) [reply code/modules/research/designs/medical_designs.dm]: Also seems as if this recipe would be better put into the autolathe.

File: code/modules/research/designs/medical_designs.dm [hunk 2]
```diff
@@ -526,6 +536,80 @@
 	category = list("Medical Designs")
 	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL
 
+/////////////////////
+//Adv Surgery Tools//
+/////////////////////
+
+/datum/design/drapes
```
Comment (Ragolution) [inline code/modules/research/designs/medical_designs.dm:543]: Drapes also really belong in the autolathe and not the protolathe

File: code/modules/research/designs/medical_designs.dm [hunk 3]
```diff
@@ -526,6 +536,80 @@
...
+	id = "drapes"
+	build_type = PROTOLATHE
+	materials = list(MAT_PLASTIC = 2500)
+	build_path = /obj/item/surgical_drapes
+	category = list("Medical Designs")
+	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL | DEPARTMENTAL_FLAG_SCIENCE
+
+/datum/design/retractor_adv
+	name = "Advanced retractor"
+	desc = "Retracts stuff. This one has been made of silver and gold, clearly making it better at retracting!"
```
Comment (Ragolution) [inline code/modules/research/designs/medical_designs.dm]:
```suggestion
	desc = "A high-class, premium retractor, featuring precision crafted, silver-plated hook-ends and an electrum handle."
```
Or whatever. Just a suggestion.

File: code/modules/research/designs/medical_designs.dm [hunk 4]
```diff
@@ -526,6 +536,80 @@
...
+	desc = "A large surgery drape made of plastic."
+	id = "drapes"
+	build_type = PROTOLATHE
+	materials = list(MAT_PLASTIC = 2500)
+	build_path = /obj/item/surgical_drapes
+	category = list("Medical Designs")
+	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL | DEPARTMENTAL_FLAG_SCIENCE
+
+/datum/design/retractor_adv
+	name = "Advanced retractor"
```
Comment (Ragolution) [inline code/modules/research/designs/medical_designs.dm]:
```suggestion
	name = "Advanced Retractor"
```

File: code/modules/research/designs/medical_designs.dm [hunk 5]
```diff
@@ -526,6 +536,80 @@
...
+	desc = "Retracts stuff. This one has been made of silver and gold, clearly making it better at retracting!"
+	id = "retractor_adv"
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 500, MAT_GLASS = 500, MAT_SILVER = 1500, MAT_GOLD = 1000)
+	build_path = /obj/item/retractor/adv
+	category = list("Medical Designs")
+	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL | DEPARTMENTAL_FLAG_SCIENCE
+
+/datum/design/hemostat_adv
+	name = "advanced hemostat"
```
Comment (Ragolution) [inline code/modules/research/designs/medical_designs.dm]:
```suggestion
	name = "Advanced Hemostat"
```

File: code/modules/research/designs/medical_designs.dm [hunk 6]
```diff
@@ -526,6 +536,80 @@
...
+	desc = "You think you have seen this before. But not as golden or lustrous as this one..."
```
Comment (Ragolution) [inline code/modules/research/designs/medical_designs.dm]:
```suggestion
	desc = "An exceptionally fine pair of arterial forceps. These appear to be plated in electrum and feel soft to the touch."
```

File: code/modules/research/designs/medical_designs.dm [hunk 7]
```diff
@@ -526,6 +536,80 @@
...
+	id = "hemostat_adv"
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 500, MAT_GLASS = 500, MAT_SILVER = 1000, MAT_GOLD = 1500)
+	build_path = /obj/item/hemostat/adv
+	category = list("Medical Designs")
+	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL | DEPARTMENTAL_FLAG_SCIENCE
+
+/datum/design/cautery_adv
+	name = "advanced cautery"
```
Comment (Ragolution) [inline code/modules/research/designs/medical_designs.dm]:
```suggestion
	name = "Electrocautery" //This is based on real-life science.
```

File: code/modules/research/designs/medical_designs.dm [hunk 8]
```diff
@@ -526,6 +536,80 @@
...
+	desc = "This stops bleeding. This cautery has a exstended nose with a golden tip at the end."
```
Comment (Ragolution) [inline code/modules/research/designs/medical_designs.dm]:
```suggestion
	desc = "This stops bleeding. This cautery has a extended nose with a golden tip at the end."
```

Fix typo

Comment (Ragolution) [reply code/modules/research/designs/medical_designs.dm]: Alternatively:
```suggestion
	desc = "A high-tech unipolar Electrocauter. This tiny device contains an extremely powerful microbattery that uses arcs of electricity to painlessly sear wounds shut. It seems to recharge with the user's body-heat. Wow!"
```

File: code/modules/research/designs/medical_designs.dm [hunk 9]
```diff
@@ -526,6 +536,80 @@
...
+	id = "cautery_adv"
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 500, MAT_GLASS = 500, MAT_SILVER = 1000, MAT_GOLD = 1500)
+	build_path = /obj/item/cautery/adv
+	category = list("Medical Designs")
+	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL | DEPARTMENTAL_FLAG_SCIENCE
+
+/datum/design/surgicaldrill_adv
+	name = "advanced surgicaldrill"
```
Comment (Ragolution) [inline code/modules/research/designs/medical_designs.dm]:
```suggestion
	name = "Advanced Surgical Drill"
```
or
```suggestion
	name = "Surgical Autodrill"
```

File: code/modules/research/designs/medical_designs.dm [hunk 10]
```diff
@@ -526,6 +536,80 @@
...
+	desc = "You can drill using this item. It has a damion tip at the end of its' golden drill."
```
Comment (Ragolution) [inline code/modules/research/designs/medical_designs.dm]:
```suggestion
	desc = "You can drill using this item. It has a diamond tip at the end of its golden drill."
```
or
```suggestion
	desc = "With a diamond tip and built-in depth and safety sensors, this drill alerts the user before overpenetrating a patient's skull or tooth. There also appears to be a disable switch."
```

File: code/modules/research/designs/medical_designs.dm [hunk 11]
```diff
@@ -526,6 +536,80 @@
...
+	id = "surgicaldrill_adv"
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 2500, MAT_GLASS = 2500, MAT_SILVER = 6000, MAT_GOLD = 5500, MAT_DIAMOND = 3500)
+	build_path = /obj/item/surgicaldrill/adv
+	category = list("Medical Designs")
+	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL | DEPARTMENTAL_FLAG_SCIENCE
+
+/datum/design/scalpel_adv
+	name = "advanced scalpel"
```
Comment (Ragolution) [inline code/modules/research/designs/medical_designs.dm]:
```suggestion
	name = "Advanced Scalpel"
```
or
```suggestion
	name = "Precision Scalpel"
```

File: code/modules/research/designs/medical_designs.dm [hunk 12]
```diff
@@ -526,6 +536,80 @@
...
+	desc = "Cut, cut, and once more cut now with a golden eage. Clearly made of mostly silver yet as you pick it up it feels lighter then normal."
```
Comment (Ragolution) [inline code/modules/research/designs/medical_designs.dm]:
```suggestion
	desc = "A perfectly balanced electrum scalpel with a silicon-coated edge to eliminate wear and tear."
```

File: code/modules/research/designs/medical_designs.dm [hunk 13]
```diff
@@ -526,6 +536,80 @@
...
+	id = "scalpel_adv"
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 1500, MAT_GLASS = 1500, MAT_SILVER = 4000, MAT_GOLD = 2500)
+	build_path = /obj/item/scalpel/adv
+	category = list("Medical Designs")
+	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL | DEPARTMENTAL_FLAG_SCIENCE
+
+/datum/design/circular_saw_adv
+	name = "advanced circular saw"
```
Comment (Ragolution) [inline code/modules/research/designs/medical_designs.dm]:
```suggestion
	name = "Diamond-Grit Circular Saw"
```

File: code/modules/research/designs/medical_designs.dm [hunk 14]
```diff
@@ -526,6 +536,80 @@
...
+	desc = "For heavy duty cutting now with a damion eage saw!"
```
Comment (Ragolution) [inline code/modules/research/designs/medical_designs.dm]:
```suggestion
	desc = "For those Assistants with REALLY thick skulls."
```

File: code/modules/research/techweb/all_nodes.dm [hunk 15]
```diff
@@ -89,6 +89,15 @@
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
 	export_price = 5000
 
+/datum/techweb_node/advance_surgerytools
+	id = "advance_surgerytools"
+	display_name = "Advanced Surgery Tools"
+	description = "New faster and lighter surgical tools made of rare metals and gems."
```
Comment (Ragolution) [inline code/modules/research/techweb/all_nodes.dm]:
```suggestion
	description = "Refined and improved redesigns for the run-of-the-mill medical utensils."
```

Comment (Ragolution) [reply code/modules/research/techweb/all_nodes.dm]: Or whatever.

**Review by Ragolution — CHANGES_REQUESTED** [review]

**Review by silicons — CHANGES_REQUESTED** [review]
most of what rags said

File: code/modules/research/designs/medical_designs.dm (hunk 14 above)
Comment (Ragolution) [reply code/modules/research/designs/medical_designs.dm]: This one is my favourite.

File: code/modules/surgery/tools.dm [hunk 16]
```diff
@@ -7,6 +7,15 @@
 	flags_1 = CONDUCT_1
 	w_class = WEIGHT_CLASS_TINY
 
+/obj/item/retractor/adv
+	name = "Advanced retractor"
```
Comment (Ragolution) [inline code/modules/surgery/tools.dm]:
```suggestion
	name = "Advanced Retractor"
```

File: code/modules/surgery/tools.dm [hunk 17]
```diff
@@ -7,6 +7,15 @@
 	flags_1 = CONDUCT_1
 	w_class = WEIGHT_CLASS_TINY
 
+/obj/item/retractor/adv
+	name = "Advanced retractor"
+	desc = "Retracts stuff. This one has been made of silver and gold, clearly making it better at retracting!"
```
Comment (Ragolution) [inline code/modules/surgery/tools.dm]:
```suggestion
	desc = "A high-class, premium retractor, featuring precision crafted, silver-plated hook-ends and an electrum handle."
```

File: code/modules/surgery/tools.dm [hunk 18]
```diff
@@ -29,6 +37,16 @@
 	w_class = WEIGHT_CLASS_TINY
 	attack_verb = list("attacked", "pinched")
 
+/obj/item/hemostat/adv
+	name = "advanced hemostat"
+	desc = "You think you have seen this before. But not as golden or lustrous as this one..."
```
Comment (Ragolution) [inline code/modules/surgery/tools.dm]:
```suggestion
	desc = "An exceptionally fine pair of arterial forceps. These appear to be plated in electrum and feel soft to the touch."
```

File: code/modules/surgery/tools.dm [hunk 19]
```diff
@@ -29,6 +37,16 @@
 	w_class = WEIGHT_CLASS_TINY
 	attack_verb = list("attacked", "pinched")
 
+/obj/item/hemostat/adv
+	name = "advanced hemostat"
```
Comment (Ragolution) [inline code/modules/surgery/tools.dm]:
```suggestion
	name = "Advanced Hemostat"
```

File: code/modules/surgery/tools.dm [hunk 20]
```diff
@@ -52,6 +70,16 @@
 	w_class = WEIGHT_CLASS_TINY
 	attack_verb = list("burnt")
 
+/obj/item/cautery/adv
+	name = "advanced cautery"
```
Comment (Ragolution) [inline code/modules/surgery/tools.dm]:
```suggestion
	name = "Electrocautery"
```

File: code/modules/surgery/tools.dm [hunk 21]
```diff
@@ -52,6 +70,16 @@
 	w_class = WEIGHT_CLASS_TINY
 	attack_verb = list("burnt")
 
+/obj/item/cautery/adv
+	name = "advanced cautery"
+	desc = "This stops bleeding. This cautery has a exstended nose with a golden tip at the end."
```
Comment (Ragolution) [inline code/modules/surgery/tools.dm]:
```suggestion
	desc = "A high-tech unipolar Electrocauter. This tiny device contains an extremely powerful microbattery that uses arcs of electricity to painlessly sear wounds shut. It seems to recharge with the user's body-heat. Wow!"
```

File: code/modules/surgery/tools.dm [hunk 22]
```diff
@@ -79,6 +107,21 @@
 	w_class = WEIGHT_CLASS_NORMAL
 	attack_verb = list("drilled")
 
+/obj/item/surgicaldrill/adv
+	name = "advanced surgical drill"
+	desc = "You can drill using this item. It has a damion tip at the end of its' golden drill."
```
Comment (Ragolution) [inline code/modules/surgery/tools.dm]:
```suggestion
	desc = "With a diamond tip and built-in depth and safety sensors, this drill alerts the user before overpenetrating a patient's skull or tooth. There also appears to be a disable switch."
```

File: code/modules/surgery/tools.dm [hunk 23]
```diff
@@ -116,6 +159,25 @@
 	. = ..()
 	AddComponent(/datum/component/butchering, 80 * toolspeed, 100, 0)
 
+/obj/item/scalpel/adv
+	name = "advanced scalpel"
```
Comment (Ragolution) [inline code/modules/surgery/tools.dm]:
```suggestion
	name = "Precision Scalpel"
```

File: code/modules/surgery/tools.dm [hunk 24]
```diff
@@ -116,6 +159,25 @@
 	. = ..()
 	AddComponent(/datum/component/butchering, 80 * toolspeed, 100, 0)
 
+/obj/item/scalpel/adv
+	name = "advanced scalpel"
+	desc = "Cut, cut, and once more cut now with a golden eage. Clearly made of mostly silver yet as you pick it up it feels lighter then normal"
```
Comment (Ragolution) [inline code/modules/surgery/tools.dm]:
```suggestion
desc = "A perfectly balanced electrum scalpel with a silicon-coated edge to eliminate wear and tear."
```

File: code/modules/surgery/tools.dm [hunk 25]
```diff
@@ -161,6 +223,26 @@
 	. = ..()
 	AddComponent(/datum/component/butchering, 40 * toolspeed, 100, 5, 'sound/weapons/circsawhit.ogg') //saws are very accurate and fast at butchering
 
+/obj/item/circular_saw/adv
+	name = "advanced circular saw"
```
Comment (Ragolution) [inline code/modules/surgery/tools.dm]:
```suggestion
	name = "Diamond-Grit Circular Saw"
```

File: code/modules/surgery/tools.dm [hunk 26]
```diff
@@ -161,6 +223,26 @@
 	. = ..()
 	AddComponent(/datum/component/butchering, 40 * toolspeed, 100, 5, 'sound/weapons/circsawhit.ogg') //saws are very accurate and fast at butchering
 
+/obj/item/circular_saw/adv
+	name = "advanced circular saw"
+	desc = "For heavy duty cutting now with a damion eage saw!"
```
Comment (Ragolution) [inline code/modules/surgery/tools.dm]:
```suggestion
	desc = "For those Assistants with REALLY thick skulls."
```

**Review by Ragolution — CHANGES_REQUESTED** [review]
Mirrors changes made to medical_designs.dm

File: code/modules/surgery/tools.dm [hunk 27]
```diff
@@ -79,6 +107,21 @@
 	w_class = WEIGHT_CLASS_NORMAL
 	attack_verb = list("drilled")
 
+/obj/item/surgicaldrill/adv
+	name = "advanced surgical drill"
```
Comment (Ragolution) [inline code/modules/surgery/tools.dm]:
```suggestion
	name = "Surgical Autodrill"
```

**Review by Ragolution — CHANGES_REQUESTED** [review]
Forgot this one.

**Comment by Ragolution** [comment]
Looks fine if @kevinz000 thinks it passes Balance.

**Review by Ragolution — APPROVED** [review]
Looks fine if @kevinz000 thinks it passes Balance.

**Review by silicons — APPROVED** [review]

**Pull request merged**